client/gettext/json/available_languages.json

```
@@ -1 +1 @@
-{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
+{"es-ES":"Español","es":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```

@mzazrivec tests are thinking this ought to look like this...  Español being in there twice looks to be a woopsie 🤔 
```suggestion
{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```